Revert change between 3.0.9rc3 and 3.0.9rc4 for pkg-config --cflags-only-I to
return correct path and to avoid gcc's directory.

--- include/Makefile.am
+++ include/Makefile.am
@@ -5,8 +5,5 @@
 DISTCLEANFILES=ffitarget.h
 EXTRA_DIST=ffi.h.in ffi_common.h
 
-# Where generated headers like ffitarget.h get installed.
-gcc_version   := $(shell cat $(top_srcdir)/../gcc/BASE-VER)
-toollibffidir := $(libdir)/gcc/$(target_alias)/$(gcc_version)/include
-
-toollibffi_HEADERS = ffi.h ffitarget.h
+includesdir = $(libdir)/@PACKAGE_NAME@-@PACKAGE_VERSION@/include
+nodist_includes_HEADERS = ffi.h ffitarget.h 
